packages/cxl-ui/src/components/cxl-playbook-accordion.js

```diff
@@ -0,0 +1,156 @@
+import { customElement } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/vaadin-accordion';
```

Comment:
Double import - already done by `CXLVaadinAccordion`

I wonder why automation doesn't detect it :thinking:

Reply:
no clue, removed in 6fca313
`import { customElement } from 'lit-element';` this has to be doubled though...